codeflash/benchmarking/pytest_new_process_trace_benchmarks.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from pathlib import Path
+
+from codeflash.benchmarking.benchmark_database_utils import BenchmarkDatabaseUtils
```

Comment:
can you not import from codeflash? it will cause errors like https://linear.app/codeflash-ai/issue/CF-436/dont-use-pytestmain-for-test-discovery